otherlibs/site/src/plugins/plugins.ml

```diff
@@ -207,7 +234,15 @@ let load_gen ~load_requires dirs name =
 let rec load_requires name =
   load_gen ~load_requires (Lazy.force Helpers.ocamlpath) name
 
-let load_plugin plugin_paths name = load_gen ~load_requires plugin_paths name
+let load_plugin plugin_paths name =
+  match lookup plugin_paths (Filename.concat name meta_fn) with
+  | None -> raise (Plugin_not_found (plugin_paths, name))
+  | Some meta_file ->
+    let meta = load meta_file ~pkg:name in
+    let plugins, requires = get_plugin [] [] meta.Meta_parser.entries in
```

Comment:
In our coding style, we add a type annotation to the name instead. So `name` should be annotated instead of the module name.

Reply:
There's no link to it, but I can assure it is the style we use in the vast majority of cases. In any case, feel free to leave it as you like, it's not a big deal :)